### PR TITLE
Add Typhoon

### PR DESCRIPTION
**Pull request by TonyLianLong**
Add typhoon to the map.
image
#10

### CODE REVIEW

**Review by dkess — APPROVED** [review]
Awesome, thanks so much!

**Comment by dkess** [comment]
This PR is actually not working-- the API can correctly show that the desktop is in use, but the desktop won't get greyed out. Perhaps you forgot to copy something over?

File: www/index.html [hunk 1]
```diff
@@ -1614,6 +1632,28 @@
...
+        </g>
+        <g>
+            <g>
+                <polygon class="st52" points="1356.8,754.7 1350.8,758.2 1350.8,789.5 1356.8,786 					"/>
+                <polygon class="desk-left" points="1349.4,757.3 1350.8,758.2 1350.8,789.5 1349.4,788.7 					"/>
+                <polygon class="st19" points="1349.4,757.3 1350.8,758.2 1356.8,754.7 1355.4,753.9 					"/>
+            </g>
+        </g>
+        <g>
+            <polygon class="comp c28 available" points="1380.8,723.2 1322.6,756.5 1322.6,797.1 1380.8,763.8 				"/>
```
Comment (cg505) [inline www/index.html:1651]:
```suggestion
            <polygon class="comp c33 available" points="1380.8,723.2 1322.6,756.5 1322.6,797.1 1380.8,763.8 				"/>
```
?

Comment (TonyLianLong) [reply www/index.html:1651]: Oh that should be `c33`. You're right. I'll open another PR.

**Comment by TonyLianLong** [comment]
#12

**Pull request merged**